ajax/libs/covjson-reader/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "covjson-reader",
+  "filename": "covjson-reader.min.js",
+  "version": "0.11.6",
+  "license": "BSD-3-Clause",
+  "description": "A reader for CoverageJSON files.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Reading-eScience-Centre/covjson-reader.git"
+  },
+  "keywords": [
+    "CoverageJSON",
+    "reader"
+  ],
+  "author": "Maik Riechert",
+  "dependencies": {
+    "covutils": "^0.2.0",
+    "cbor-js": "^0.1.0",
+    "ndarray": "^1.0.18",
+    "request-promise": "^1.0.2"
+  },
+  "homepage": "https://doc.esdoc.org/github.com/reading-escience-centre/covjson-reader/",
```

Comment:
Did you add this by yourself? I don't think the api document === homepage of a project.